website/pages/docs/plugins/sources/tailscale/_meta.json

```diff
@@ -1,5 +1,4 @@
 {
   "overview": "Overview",
-  "configuration": "Configuration",
```

Comment:
Can we add a redirect from `/docs/plugins/sources/tailscale/configuration` to `/docs/plugins/sources/tailscale/overview` in `vercel.json`?